README.md

```diff
@@ -1,5 +1,13 @@
-# launch31-group-project
+# launch31-group-project: Podcasts Review App
 
-Meme team Dream team
+-- Description -- This application displays various podcast genres, their correlating podcasts, and reviews for those podcasts. It also allows users to add podcasts and reviews of their own.
 
```

Comment:
Great start here! Take a look at the markdown syntax guide, the hashtags for headers looks good but we don't need all of these dashes

https://www.markdownguide.org/basic-syntax